tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "es2018",
```

Comment:
I needed to change this because without it I would get this error: 

```
addon-test-support/audit.ts:89:6 - error TS2550: Property 'finally' does not exist on type 'Promise<void>'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2018' or later.
```

I don't know what was making this pass before 🤷